Makefile

```diff
@@ -3,6 +3,9 @@
 test:
 	pytest --cov=great_tables --cov-report=xml
 
+test-no-pandas:
+	pytest tests/test_dependencies.py -m "no_pandas"
```

Comment:
we only test this one file, because other tests do top-level imports of pandas. So if you don't have pandas installed, the tests would fail on collection.